content/en/platform/corda/1.3/cenm/deployment-kubernetes.md

```diff
@@ -129,24 +129,22 @@ The deployment steps are given below:
 - Connect to [your cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal#connect-to-the-cluster)
   from your local machine.
 
-#### 3. Create storage class and namespace
+#### 3. Download CENM deployment scripts
 
-Run the following instruction once the previous points have been cleared:
+You can find the files required for the following steps in [CENM deployment repo](https://github.com/corda/cenm-deployment).
+
+#### 4. Create storage class and namespace
 
-`All the examples below use the namespace **cenm**`
+Run the following instruction once the previous points have been cleared. These examples use the namespace **cenm**:
 
 ```bash
-kubectl apply -f deployment/k8s/cenm.yaml
+kubectl apply -f ./k8s/cenm.yaml
```

Comment:
Pedantic - perhaps we could do away with the "./" here to be consistent with other paths on the page? We seem to have a mix in other files as well, maybe have `k8s/<something>` throughout?

Reply:
Also don't we need a note so say replace "[use your own group]" with the resource group name?

Reply:
I've raised this ticket, as not sure what the change should be in response to Chris' comment: https://r3-cev.atlassian.net/browse/DOC-3905